integration-templates/zoho-crm/zoho-crm-deals.ts

```diff
@@ -0,0 +1,114 @@
+import type { ZohoCRMDeal, NangoSync } from './models';
+
+export default async function fetchData(nango: NangoSync) {
+    try {
+        const responses = await getAllDeals(nango, '/crm/v2/Deals');
```

Comment:
Same here. We need to save page per page. Ideally using the `nango.paginate` notation.

Reply:
Hi @bastienbeurier thanks for the insight, I am working on the above fixes and will be submitting in a few.